tests/unit/s2n_server_hello_test.c

```diff
@@ -154,6 +155,51 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(server_conn));
     }
 
+    /* Test Server Hello Recv with invalid cipher */
+    {
+        struct s2n_config *server_config;
+        struct s2n_config *client_config;
```

Comment:
It doesn't look like you're changing anything on these configs. If you create a connection, it will use the default config, which should be fine here.

Reply:
Removed both configs